fontreport/fontreport.py

```diff
@@ -606,14 +606,14 @@ def main():
         tofile = name + '.tex'
 
     with open(tofile, 'w') as f:
-      f.write(envelope.Report(xetex).encode('utf-8'))
+      f.write(envelope.Report(xetex))
     if ext == '.pdf':
       # Call twice to let longtable package calculate
       # column width correctly
       subprocess.check_call(['xelatex', tofile])
       subprocess.check_call(['xelatex', tofile])
   else:
-    print envelope.Report(False).encode('utf-8')
+    print(envelope.Report(False))
```

Comment:
Implicit encoding is 'ascii' when redirecting fontreport data to other program. It results in exception if a report contain any character that does not fit into 0-127 range.